src/utils/validIdList.js

```diff
@@ -0,0 +1,21 @@
+const validIdListRegex = /(([A-Za-z]+[\w\-\:\.]*)\s*)*/;
```

Comment:
would be good to add tests for this util, with a variety of different test cases